L1Trigger/L1TMuonOverlapPhase1/src/Tools/CandidateSimMuonMatcher.cc

```diff
@@ -101,8 +104,8 @@ bool simTrackIsMuonInOmtf(const SimTrack& simTrack) {
 
   //higher margin for matching must be used than actual OMTF region (i.e. 0.82 - 1.24),
   //otherwise many candidates are marked as ghosts
-  //if( (fabs(simTrack.momentum().eta()) >= 0.82 ) && (fabs(simTrack.momentum().eta()) <= 1.24) ) {
-  if ((fabs(simTrack.momentum().eta()) >= 0.72) && (fabs(simTrack.momentum().eta()) <= 1.3)) {
+  //if( (std::abs(simTrack.momentum().eta()) >= 0.82 ) && (std::abs(simTrack.momentum().eta()) <= 1.24) ) {
```

Comment:
```suggestion
```
I can't really tell whether this file was designed to work in general CMSSW operation, so for the next few comments, I think you can ignore some of the picky stuff from me if it isn't.

Reply:
It is indeed not designed in general CMSSW operation, i.e. in standard emulation workflow. It is a tool which matches simulated muon with OMTF response (from emulator) used for test or preparing training samples for NN. 
Anyway, thank you for your comments we will try to address them to improve quality of this code.